scripts/ci/pre_commit/update_providers_build_files.py

```diff
@@ -85,10 +60,7 @@ def _find_new_providers_structure() -> None:
 
 # get all folders from arguments
 for examined_file in file_list:
-    if not examined_file.startswith("providers/src"):
-        _find_new_providers_structure()
-    else:
-        _find_old_providers_structure()
+    _find_new_providers_structure(Path(examined_file))
```

Comment:
I will find and replace "_new_providers"  and remove all other "_old_providers" in a follow-up PR.

Reply:
Nit: Can you add a todo then into code?

Reply:
I will have the PR VERY soon after merging this one. So no worries it will be gone in no time at all.